Formula/ocaml-findlib.rb

```diff
@@ -0,0 +1,25 @@
+class OcamlFindlib < Formula
+  desc "OCaml library manager"
+  homepage "http://projects.camlcity.org/projects/findlib.html"
+  url "http://download.camlcity.org/download/findlib-1.7.3.tar.gz"
+  sha256 "d196608fa23c36c2aace27d5ef124a815132a5fcea668d41fa7d6c1ca246bd8b"
+
+  depends_on "ocaml"
+
+  def install
+    ENV.deparallelize
```

Comment:
needs to be fixed upstream

Reply:
This doesn't appear to be needed. It seems to build fine without it. Were you seeing build failures?

Reply:
I guess I don't understand the semantics of `ENV.deparallelize`. Clearly, `./configure` has to happen first, and `make install` has to happen last.

Reply:
ENV.deparallelize has nothing to do with configure. It sets the number of make jobs to 1. So make becomes make -j1. That isn't needed here.

Reply:
FYI: It turns out the Makefile is indeed broken. I submitted a patch upstream. See https://gitlab.camlcity.org/gerd/lib-findlib/merge_requests/8 .